### ISSUE TITLE
rangeDataItem

### ISSUE THREAD

**Issue by SekMENtina** [amcharts/amcharts5#18]
Hi!
It seems 
```
rangeDataItem.get("axisFill").setAll({
    fill: color,
    fillOpacity: 0.5
  });
```
doesnt work
https://codepen.io/team/amcharts/pen/XWRoPVB

**Comment by martynasma** [comment]
Fixed in [5.0.0-beta.25](https://github.com/amcharts/amcharts5/releases/tag/5.0.0-beta.25).

## [5.0.0-beta.25] - 2021-09-24

### Added
- New `Entity` setting: `userData`. Use it to store any kind of proprietary data with any object in amCharts 5 that extends `Entity`.

### Changed
- Series axis range will now have `visible: false` set for all axis elements (grid, fill, label, tick) by deafult. If you need to enable them, set `visible: true` on a respective range element. [More info](https://www.amcharts.com/docs/v5/charts/xy-chart/axes/axis-ranges/#Elements_on_a_series_range).

### Fixed
- Improperly closed in-line text style blocks could cause label palcement issues.
- Using `\n\n` in text was not resulting in empty line.
- Horizontal `Sankey` bullets were not using their `locationX` as it should.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download-v5/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by martynasma** [comment]
Please note that from this release you will need to explicitly enable series axis range elements like fill, as they are not visible by default:

```JavaScript
rangeDataItem.get("axisFill").setAll({
  fill: color,
  fillOpacity: 0.5,
  visible: true
});
```
